### PR TITLE
RHDEVDOCS-3424 - Remove Prometheus UI references from Monitoring documentation

### PR DESCRIPTION
**Pull request by bburt-rh**
- Aligned team: DevTools
- For branches: 4.10+
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3424
- Direct link to doc preview: https://deploy-preview-38652--osdocs.netlify.app/openshift-enterprise/latest/monitoring/accessing-third-party-uis
- SME review: @kyoto 
- QE review: @juzhao and @lihongyan1 
- Peer review: @ tbd
- <All reviews complete. Please merge now>

Note: Removal of references to other third-party UIs are being done in separate PRs

### CODE REVIEW

**Comment by kyoto** [comment]
LGTM

File: monitoring/accessing-third-party-uis.adoc [hunk 1]
```diff
@@ -5,32 +5,13 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-Integrated Metrics, Alerting, and Dashboard UIs are provided in the {product-title} web console. See the following for details on using these integrated UIs:
+{product-title} does not provide or support direct access to third-party web user interfaces (UIs) for components in the Monitoring stack such as Prometheus, Alertmanager, or Thanos Querier.
```
Comment (juzhao) [inline monitoring/accessing-third-party-uis.adoc:8]: missed Grafana

**Comment by juzhao** [comment]
@lihongyan1 PTAL

**Comment by juzhao** [comment]
I think this PR is also for

RHDEVDOCS-3426 | Alertmanager UI removal notice in Monitoring documentation
RHDEVDOCS-3425 | Thanos UI removal notice in Monitoring documentation
RHDEVDOCS-3389 | Grafana removal notice in Monitoring documentation

**Comment by bburt-rh** [comment]
> I think this PR is also for
> 
> [RHDEVDOCS-3426](https://issues.redhat.com/browse/RHDEVDOCS-3426) | Alertmanager UI removal notice in Monitoring documentation [RHDEVDOCS-3425](https://issues.redhat.com/browse/RHDEVDOCS-3425) | Thanos UI removal notice in Monitoring documentation [RHDEVDOCS-3389](https://issues.redhat.com/browse/RHDEVDOCS-3389) | Grafana removal notice in Monitoring documentation

Doing those issues as separate PRS. The Grafana removal affects a lot of other files, for instance.

File: monitoring/accessing-third-party-uis.adoc (hunk 1 above)
Comment (bburt-rh) [reply monitoring/accessing-third-party-uis.adoc]: Fixed. Added Grafana.

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**